MIGRATION.md

```diff
@@ -0,0 +1,35 @@
+# Migration guide
+
+## From 0.x.x to 1.0.0
+
+On the backend side the changes are simple:
+
+1. Remove the `AsyncRequest::ApplicationHelper` from your controller.
+2. The gem now does not expect a JSON as the result of the worker but any a string, so you now must perform a `.to_json` to the result of the worker:
```

Comment:
you MUST ? Always ? Why do we change it then ? I would say you must if you are rendering a json right ?

Reply:
@Dinuuu about the last comment, since this is a migration guide, i think it's ok to asume that you were rendering json, anyways if some of this text goes to the main readme i'll agree to add explicitly say that